apps/web/package.json

```diff
@@ -62,6 +63,7 @@
     "@stripe/react-stripe-js": "^1.10.0",
     "@stripe/stripe-js": "^1.35.0",
     "@tanstack/react-query": "^5.17.15",
+    "@teamhanko/passkeys-next-auth-provider": "^0.2.7",
```

Comment:
I'm not sure since when npm has this, but there is **optionalDependencies**

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#optionaldependencies

Was supported in yarn v1 so I assume v2+ have it too

Reply:
Opted not to do this for now because optionalDependencies isn't in use for any other packages atm. Also I'm unsure if a build without installing optional deps would be successful, especially because of `import` behavior, haven't tested it

Do you want me to test this? What other not-strictly-needed dependencies are there that could be moved to `optionalDependencies` (at least in theory)?